### ISSUE TITLE
Could not find a declaration file for module 'react-layers-manager'. 

### ISSUE THREAD

**Issue by lps83** [giuseppeg/react-layers-manager#7]
Hi, I installed the plugin through NPM, but sadly it's impossible to use it.

Could not find a declaration file for module 'react-layers-manager'. '.../node_modules/react-layers-manager/lib/index.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/react-layers-manager` if it exists or add a new declaration (.d.ts) file containing `declare module 'react-layers-manager';`ts(7016)

Any idea ? Thanks

**Comment by giuseppeg** [comment]
yeah I don't have types for the package. You may want to try add it to the exclude paths in your tsconfig.json

**Comment by leoweyr** [comment]
Done with #8.
